packages/core/src/command/patient-import/commands/check-patient-record.ts

```diff
@@ -0,0 +1,48 @@
+import { errorToString } from "@metriport/shared";
+import { S3Utils } from "../../../external/aws/s3";
+import { out } from "../../../util/log";
+import { capture } from "../../../util/notifications";
+import { Config } from "../../../util/config";
+import { createFileKeyPatient } from "../patient-import-shared";
+
+const region = Config.getAWSRegion();
+
+function getS3UtilsInstance(): S3Utils {
+  return new S3Utils(region);
+}
+
+export async function checkPatientRecord({
```

Comment:
for later: I'd name this something else, this seems it checks for validity, not whether it has already been processed

Reply:
Done.